docs/reference/webhooks-reference.md

```diff
@@ -1,5 +1,173 @@
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`   | **str / required**<br/>The webhook name.                                                                                                                                                    |
+| `url`    | **str / required**<br/>A unique URL where the webhook will send the message. HTTPS is required.                                                                                             |
+| `events` | **list[str]**<br/>A list of events that trigger the webhook. The allowed values:<br/><ul><li>`job.status`</li><li>`order.status`</li></ul>                                                  |
+| `active` | **bool**<br/>Determines whether the webhook is active after creation:<br/><ul><li>`True`: webhook is active.</li><li>`False`: webhook is not active.</li></ul>The default value is `False`. |
+| `secret` | **optional[str]**<br/>The secret used to generate webhook signatures. The default value is `None`.                                                                                          |
+
+<h5> Example </h5>
+
+```python
+webhook = up42.create_webhook(
+    name="new-webhook",
+    url="https://receiving-url.com",
+    events=["job.status", "order.status"],
+    active=True,
+    secret="QWZTFnMEXhqZKNmu",
+)
+```
+
+### update()
+
+The `update()` function allows you to modify a specific webhook.
+
+```python
+update(
+    name,
+    url,
+    events,
+    active,
+    secret,
+)
+```
+
+The returned format is `dict`.
+
+<h5> Arguments </h5>
+
+| Argument | Overview                                                                                                                                                      |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`   | **str**<br/>An updated webhook name.                                                                                                                          |
+| `url`    | **str**<br/>An updated unique URL where the webhook will send the message. HTTPS is required.                                                                 |
+| `events` | **list[str]**<br/>An updated list of events that trigger the webhook. The allowed values:<br/><ul><li>`job.status`</li><li>`order.status`</li></ul>           |
+| `active` | **bool**<br/>An updated webhook status:<br/><ul><li>`True`: webhook is active.</li><li>`False`: webhook is not active.</li></ul>The default value is `False`. |
+| `secret` | **optional[str]**<br/>An updated secret used to generate webhook signatures. The default value is `None`.                                                     |
```

Comment:
An updated secret used to generate webhook signatures. -> The secret used to generate webhook signatures.